### ISSUE TITLE
Freeze after channel scan with no result

### ISSUE THREAD

**Issue by AlbrechtL** [AlbrechtL/welle.io#21]
When search and no reception results, no rescan is possible and application freezes.

**Comment by tnadolecki** [comment]
Does not seem to matter whether there were stations found or not. Freezes at every second scan attempt.

**Comment by AlbrechtL** [comment]
Which operating system do you are using?

**Comment by tnadolecki** [comment]
I've experienced this on windows and linux. I'm totally back on linux however now.

**Comment by tnadolecki** [comment]
Please see the video I recorded for this purpose.
https://youtu.be/NVtLX1dz6TE

**Comment by AlbrechtL** [comment]
It is possible that you post screen shots from this problem? The youtube videos doesn't exist.

**Comment by tnadolecki** [comment]
https://www.youtube.com/watch?v=NVtLX1dz6TE
[https://www.youtube.com/watch?v=NVtLX1dz6TE](url)

**Comment by AlbrechtL** [comment]
The  "No channel in index"  is not the problem. Can you tell me  a little bit more about your system, please (Linux version, QT version, RTL-SDR dongle, CPU)?

**Comment by tnadolecki** [comment]
QT I only downloaded from their website 2 days ago. I don't know how to check the version, but it must be the latest.
RTL-SDR is nooelec with high stability TCXO.

0:  Realtek, RTL2838UHIDIR, SN: 00000001
Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner

CPU is DELL Laptop E6410 running Ubuntu 16.04.2 LTS

I was experiencing same behavior when in Windows 10. It would simply freeze also.
